README.md

```diff
@@ -1,14 +1,14 @@
-# JIRA REST API Client
+# Jira REST API Client
```

Comment:
Please revert this change, because in here it's library name, not Jira application itself.